### ISSUE TITLE
Can't abandon claim via app if supports added by claim owner or claim is tipped

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-app#625]
## The Issue
If you tip or support your own claim, you can no longer abandon it via the LBRY app.  Also, you run into the same scenario if someone else tips your claim (since this is now your support). The app currently uses the claim_id option to abandon the claim, and because the claim_id is the same on the tip/supports, it's not able to abandon it.  It should be using the txid/nout option of claim_abandon.  

Deamon shows: `2017-09-26 09:20:10,289 ERROR    lbrynet.daemon.auth.server:353: Failed to process claim_abandon: Abandon of fd22ecd2cba886a12a2cc6668ea421210f292557/None:None failed: more than one claim that matches`

Optimally, it should also alert the user that there are supports/tips and allow them to abandon them all at the same time. 

### Steps to reproduce
1. Create claim
2. Add tip to own claim
3. Try to abandon claim - file disappears from published list until reload, but claim not abandoned

### Expected behaviour
Show allow user to abandon all tips/supports along with the claim

### Actual behaviour
Cannot abandon

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

**Comment by kauffj** [comment]
@tzarebczan are you able to do this via the command line? Can you post the API call that fixes this, if so? If there is not an API call that allows abandoning these, then issue needs to be moved or opened in parallel on `lbry`.

**Comment by tzarebczan** [comment]
yes, there is already a call to abandon by txid/nout - https://lbryio.github.io/lbry/cli/#claim_abandon 

The discussion we had with @jackrobison on slack would be more directly related to https://github.com/lbryio/lbry-app/issues/424 - I'll open an ssue on lbry open for the enhancement.

**Comment by tzarebczan** [comment]
Per discussion with @kauffj / @hackrush01 / @jackrobison , the app side should not be in charge of figuring out the support claims, leave that to the daemon. I did open https://github.com/lbryio/lbry/issues/941 which could be used as the issue to track the daemon side changes. lbryum will also need to be enhanced. If it doesn't make sense to try to group all the UTXO from the users' claim+supports, then they can be done separately. 

The daemon needs to figure out which UTXOs belong to the user and abandon them with the claim. There would be no reason to keep the supports if the claim ID is no longer valid.

**Comment by kauffj** [comment]
Marked as blocked and moving to bottom

**Comment by tzarebczan** [comment]
daemon doesn't error anymore, but the console shows this when trying to remove from the claim page. You can remove it from the tx list
```
bundle.js:9298 Uncaught ReferenceError: txid is not defined
    at file:///C:/Program%20Files%20(x86)/LBRY/resources/app/dist/js/bundle.js:9298:14
    at file:///C:/Program%20Files%20(x86)/LBRY/resources/app/dist/js/bundle.js:29795:20
    at Array.map (native)
    at file:///C:/Program%20Files%20(x86)/LBRY/resources/app/dist/js/bundle.js:29794:45
    at file:///C:/Program%20Files%20(x86)/LBRY/resources/app/dist/js/bundle.js:115490:16
    at dispatch (file:///C:/Program%20Files%20(x86)/LBRY/resources/app/dist/js/bundle.js:47968:18)
    at file:///C:/Program%20Files%20(x86)/LBRY/resources/app/dist/js/bundle.js:9323:5
    at file:///C:/Program%20Files%20(x86)/LBRY/resources/app/dist/js/bundle.js:115487:18
    at deleteFile (file:///C:/Program%20Files%20(x86)/LBRY/resources/app/dist/js/bundle.js:123589:7)
    at onConfirmed (file:///C:/Program%20Files%20(x86)/LBRY/resources/app/dist/js/bundle.js:123678:20)
```

**Comment by tzarebczan** [comment]
as a workaround, you can abandon from the tx list. 

If there's a quick fix for the claim page side without going into https://github.com/lbryio/lbry/issues/941, would be awesome to get in.

**Comment by tzarebczan** [comment]
This is fixed now as the app uses txid + nout to abandon the claims, but I'll open a new issue to provide better notification/action when abandoning a claim that has self-supports or tips associated with it.
